tests/subresource_api_test.go

```diff
@@ -432,8 +433,20 @@ var _ = Describe("[sig-compute]Subresource Api", decorators.SigCompute, func() {
 			)
 
 			BeforeEach(func() {
-				instancetype = newVirtualMachineInstancetype(nil)
-				instancetype.Spec.CPU.Guest = 2
+				instancetype = &instancetypev1beta1.VirtualMachineInstancetype{
```

Comment:
https://github.com/kubevirt/kubevirt/pull/10456/commits/b6c87f80179c42e12ff2d66009ed15a3bcb0e5df

> These helpers will be moved into a new libinstancetype package shortly
> so instead of importing the instancetype tests here just inline the
> creation of these resources for now.